website/sidebars.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   docs: {
     Introduction: [
       'introduction/getting-started',
+      'introduction/quick-start',
```

Comment:
The placement of this page is a bit awkward. Are we assuming that the end users are going to directly navigate to the "Quick Start" page rather than reading the "Getting Started" page first? Since both of these pages start out with the same information, and have a lot of duplicate information.

Also a bit strange that we go over installation in this page, but then the next page the user might navigate to is the "Installation" page.

Reply:
Yeah, agree that there's overlap, and that needs to be sorted out.  For now, my focus is on the actual content of this page.